app/lib/actions/katello/host/hypervisors_update.rb

```diff
@@ -40,7 +40,10 @@ def load_resources
 
         def load_hosts_by_uuid
           hosts_by_uuid = ::Host.eager_load(:subscription_facet).where(katello_subscription_facets: { uuid: hypervisors_field(:uuid) })
-          @hosts.merge(hosts_by_uuid.map { |host| [host.subscription_facet.uuid, host] }.to_h)
```

Comment:
was this a no-op previously?

Reply:
It didn't update @hosts, so pretty much yeah